x/cdp/abci.go

```diff
@@ -20,6 +20,16 @@ func BeginBlocker(ctx sdk.Context, req abci.RequestBeginBlock, k keeper.Keeper)
 
 	params := k.GetParams(ctx)
 
+	// only run CDP Begin Blocker every `BeginBlockerExecutionBlockInterval` blocks
+	blockHeight := ctx.BlockHeight()
+
+	if blockHeight%params.BeginBlockerExecutionBlockInterval != 0 {
+		ctx.Logger().Info("skipping x/cdp begin blocker")
```

Comment:
We don't need info logs for this -- let's switch to Debug or remove.  Validators & exchanges already think kava/tendermint/cosmos has too many logs